src/targets/__tests__/maven.test.ts

```diff
@@ -121,17 +123,18 @@ describe('publish', () => {
 
     const revision = 'r3v1s10n';
     await mvnTarget.publish('1.0.0', revision);
-    expect(gradlePropsMock).toHaveBeenCalledTimes(1);
+    expect(createGradlePropsMock).toHaveBeenCalledTimes(1);
     expect(uploadMock).toHaveBeenCalledTimes(1);
     expect(uploadMock).toHaveBeenLastCalledWith(revision);
     expect(retrySpawnProcess).toHaveBeenCalledTimes(1);
     expect(retrySpawnProcess).toHaveBeenCalledWith(DEFAULT_OPTION_VALUE, [
```

Comment:
Shouldn't we also add a check for `deleteGradleProps` to ensure it is deleting the correct file?